lib/Features/Paypal/Controller/API/Validators/TranslatorsWhitelistAccessValidator.php

```diff
@@ -25,19 +25,26 @@ public function validate() {
 
         $user    = $this->controller->getUser();
         $project = $this->controller->getProject();
-        $membership_dao = new MembershipDao;
-
-        if (! $membership_dao->findTeamByIdAndUser( $project->id_team, $user ) ) { // not is in team
-
-            $metadata_dao = new Projects_MetadataDao;
-            $metadata     = $metadata_dao->get( $project->id, Constants::PAYPAL_WHITELIST_KEY );
-            $metadata_value = json_decode($metadata->value);
 
-            if ( !in_array( $user->getEmail(), $metadata_value->emails ) ) { // not is in whitelist
-                throw new AuthenticationError( "Nein" );
+        $membership_dao = new MembershipDao;
+        /*
+         * $project is null only when controller is supposed to handle multiple projects,
+         * in the other cases the user must to be validated when bumps into a project
+         */
+        if($project != null){
+            if (! $membership_dao->findTeamByIdAndUser( $project->id_team, $user ) ) { // not is in team
+
+                $metadata_dao = new Projects_MetadataDao;
+                $metadata     = $metadata_dao->get( $project->id, Constants::PAYPAL_WHITELIST_KEY );
+                $metadata_value = json_decode($metadata->value);
+
+                if ( !in_array( $user->getEmail(), $metadata_value->emails ) ) { // not is in whitelist
+                    throw new AuthenticationError( "Nein" );
```

Comment:
Sorry about that, _Nein_ was a joke, i wrote it only for test. Please correct, this is my fault.